src/test/java/org/pentaho/metadata/automodel/PhysicalTableImporterTest.java

```diff
@@ -30,9 +30,9 @@
 import org.pentaho.metadata.model.SqlPhysicalTable;
 
 import java.util.List;
+import java.util.Map;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
The current checkstyle rules don't check static import statements, but I was told it was best to still follow the rule of not using `*` imports... Not that it matters. 😋